api/src/main/java/org/apache/iceberg/Accessors.java

```diff
@@ -76,6 +76,20 @@ public String toString() {
     }
   }
 
+  /**
```

Comment:
Since you are describing both Position2Accessor and Position3Accessor, I think it makes more sense to move this documentation up to the `public class Accessors`, so that javadoc can be generated.